# Scorecard — a quantitative teardown 🔬
### The F-score long-short · the inverted sign · the small-cap-value / crowding explanation

![Signal: Weak](https://img.shields.io/badge/Signal-Weak-dab617?style=flat-square)
![Tradability: Mirage](https://img.shields.io/badge/Tradability-Mirage-c0392b?style=flat-square)
![F-score sorts large-cap winners?: Not supported](https://img.shields.io/badge/F--score_sorts_large--cap_winners%3F-Not_supported-8b949e?style=flat-square)

The deep companion to the [notebook for the curious](01_for_the_curious.ipynb). A celebrated accounting screen that doesn't survive on the tradable blue-chip universe.

> ⚠️ **Not investment advice.** Nine SEC EDGAR concepts + Yahoo annual returns, ~237 current S&P 500 members, XBRL era. Sources in [`docs/references.md`](../docs/references.md).

In [1]:
import sys, os
sys.path.insert(0, os.path.abspath("../../.."))  # repo root (quantlab/)
sys.path.insert(0, os.path.abspath(".."))        # study package (scorecard/)
%matplotlib inline
import matplotlib.pyplot as plt
plt.rcParams["figure.figsize"] = (10, 5.5); plt.rcParams["axes.grid"] = True
import numpy as np, pandas as pd
from scorecard import data, strategy as st
sig, fwd = data.fetch_panel()                  # cache-first (shared EDGAR pull)
h = st.quantile_hedge(sig, fwd, q=0.3, long_high=True)   # long high-F, short low-F


## Verdict, up front

| Axis | Stamp | Why |
|---|---|---|
| Signal | **Weak** | hedge −3.7%/yr, Sharpe −0.39, t −1.7 — inverted |
| Tradability | **Mirage** | low-F beat high-F on large caps |
| Sorts large-cap winners? | **Not supported** | it's a small-cap value effect |

> 💡 *In plain words:* a genuine quality screen, measured in the wrong universe.

## 1 · The claim, steelmanned

- **H₁:** high F-score predicts high forward returns.
- **H₂:** the long-high / short-low hedge is positive and significant.
- **H₃:** it's tradable on liquid large caps (not just small-cap value).

## 2 · So what? — what rides on each

H₁/H₂ would make the F-score a free, transparent quality factor straight from 10-Ks. H₃ decides whether it survives where you can trade in size. A small-cap-value-only effect is real science but useless to a large-cap book.

## 3 · How we'd know — the protocol

Nine EDGAR concepts → the 0–9 F-score → annual tercile sort → long top (healthy), short bottom (weak) → next-year hedge, Sharpe, t-stat, hit rate over the XBRL window.

## 4 · The teardown

### 4.1 The long-short, year by year

In [2]:
disp=h.copy(); disp.columns=['high-F leg','low-F leg','hedge (high-low)']
display((disp*100).round(1))
s=st.summary(h['hedge']); print({k:round(v,3) for k,v in s.items()})

,high-F leg,low-F leg,hedge (high-low)
2008,33.4,34.7,-1.3
2009,19.5,31.7,-12.2
2010,7.2,2.7,4.5
2011,19.5,25.3,-5.8
2012,37.5,46.8,-9.2
2013,23.4,16.6,6.8
2014,5.1,10.2,-5.1
2015,17.9,19.4,-1.6
2016,28.0,33.1,-5.0
2017,-3.2,5.1,-8.4


{'mean': -0.037, 'vol': 0.094, 'sharpe': -0.395, 'tstat': -1.675, 'hit_rate': 0.333, 'max_drawdown': -0.532, 'n': 18}


> 💡 *In plain words:* the hedge mean is **negative** (−3.7%/yr) and wins only ~33% of years. **H₁ and H₂ are rejected on this universe** — the sign is backwards.

### 4.2 Why it doesn't replicate — H₃ fails

Four forces flatten and flip it on large caps:

1. **It's a small-cap value effect.** Piotroski (2000) built the F-score *within* high book-to-market firms — small, neglected, distressed. That's where the spread lives.
2. **High-F is crowded.** On heavily-covered large caps a strong balance sheet is already in the price — no neglect premium left.
3. **Low-F snaps back.** Beaten-down large-cap cyclicals / troughing growth rebound, so the *short* leg wins.
4. **Survivorship.** Current members exclude the low-F firms that actually went bankrupt — the ones the short was meant to catch. **H₃ rejected.**

## 5 · The verdict

H₁/H₂/H₃ all rejected on tradable large caps → Signal `WEAK` (inverted), Tradability `MIRAGE`, sorts-large-cap-winners `NOT SUPPORTED`. The fingerprinted run is in [`docs/results.md`](../docs/results.md).

## 6 · Could you trade it?

Not on this universe. To harvest the F-score premium you'd run it on a small-cap, high-book-to-market book — illiquid, under-covered names where a clean scorecard is a genuine surprise. Outside what a free EDGAR pull on current S&P members offers.

## 7 · Going further

Forks: (a) the F-score *within* a value screen (its native habitat); (b) a Russell 2000 universe; (c) the F-score *change* vs level; (d) combine with accruals ([52](../../52-smoke-screen/)) into a composite quality factor. Backlog: [`docs/pwb_strategies_inventory.md`](../../../docs/pwb_strategies_inventory.md).